wgengine/wgengine.go

```diff
@@ -45,31 +58,55 @@ func (rs *RouteSettings) OnlyRelevantParts() string {
 		rs.LocalAddr, rs.DNS, rs.DNSDomains, peers)
 }
 
+// Router is the TODO.
```

Comment:
Hmm. Router is a <something> that is responsible for managing the system route table.

There's only one instance, and one per-OS implementation. So you could say the interface is a little bit unnecessary. But it's a nice place to hang docs.